clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -100,33 +95,29 @@ export const TcfOverlay = ({
   const [experience, setExperience] = useState<PrivacyExperience>();
 
   useEffect(() => {
-    // We need to track if the experience and GVL translations are loading separately
-    // because they are loaded asynchronously and we need to know when both are done
-    // in order to set the i18n loading state.
-    let isExperienceLoading = false;
```

Comment:
This was causing the globe to spin unnecessarily while GVL translations loaded in the background. GVL is only needed for Modal, so this is overkill on the banner.